tutorials/text/text_intro.py

```diff
@@ -31,36 +32,38 @@
 ===================
 
 The following commands are used to create text in the pyplot
-interface
+interface and the object-oriented API:
 
-* :func:`~matplotlib.pyplot.text` - add text at an arbitrary location to the ``Axes``;
-  :meth:`matplotlib.axes.Axes.text` in the API.
+* `matplotlib.pyplot.text` - add text at an arbitrary location to the Axes;
+  `matplotlib.axes.Axes.text` in the API.
 
-* :func:`~matplotlib.pyplot.xlabel` - add a label to the x-axis;
-  :meth:`matplotlib.axes.Axes.set_xlabel` in the API.
+* `matplotlib.pyplot.xlabel` - add a label to the x-axis;
+  `matplotlib.axes.Axes.set_xlabel` in the API.
 
-* :func:`~matplotlib.pyplot.ylabel` - add a label to the y-axis;
-  :meth:`matplotlib.axes.Axes.set_ylabel` in the API.
+* `matplotlib.pyplot.ylabel` - add a label to the y-axis;
+  `matplotlib.axes.Axes.set_ylabel` in the API.
 
-* :func:`~matplotlib.pyplot.title` - add a title to the ``Axes``;
-  :meth:`matplotlib.axes.Axes.set_title` in the API.
+* `matplotlib.pyplot.title` - add a title to the `~matplotlib.axes.Axes`;
+  `matplotlib.axes.Axes.set_title` in the API.
 
-* :func:`~matplotlib.pyplot.figtext` - add text at an arbitrary location to the ``Figure``;
-  :meth:`matplotlib.figure.Figure.text` in the API.
+* `matplotlib.pyplot.figtext` - add text at an arbitrary location to the
+   Figure; `matplotlib.Figure.text` in the API.
 
-* :func:`~matplotlib.pyplot.suptitle` - add a title to the ``Figure``;
-  :meth:`matplotlib.figure.Figure.suptitle` in the API.
+* `matplotlib.pyplot.suptitle` - add a title to the `~matplotlib.Figure`;
+  `matplotlib.Figure.suptitle` in the API.
 
-* :func:`~matplotlib.pyplot.annotate` - add an annotation, with
-   optional arrow, to the ``Axes`` ; :meth:`matplotlib.axes.Axes.annotate`
+* `matplotlib.pyplot.annotate` - add an annotation, with
+   optional arrow, to the `~matplotlib.axes.Axes` ; `matplotlib.axes.Axes.annotate`
    in the API.
 
 All of these functions create and return a
-:func:`matplotlib.text.Text` instance, which can be configured with a
+`matplotlib.text.Text` instance, which can be configured with a
 variety of font and other properties.  The example below shows all of
-these commands in action.
+these commands in action, and more detail is provided in the sections that
+follow.
 """
 
+import matplotlib
```

Comment:
below (unicode example): replace \374 with either the proper unicode character or "\N{LATIN SMALL LETTER U WITH DIAERESIS}" which is much more self-explanatory I think (if a bit lengthy)... (see unicodedata.name).